docs/topics/install/docker.rst

```diff
@@ -10,83 +10,78 @@ development environment.
 First you'll need to install docker_, please check the information for
 the installation steps specific to your operating system.
 
-.. note::
-    Docker recommends installing docker-toolbox_ if you're on OS X or
-    windows and that will provide you with the ``docker-machine`` and
-    ``docker-compose`` (Mac-only).
-
+There are generally two options for running docker depending on the platform
+you are running.
 
-.. _creating-the-docker-vm:
+ * Run docker on the host machine directly (recommended if supported)
```

Comment:
It's basically always supported, with the advent of Docker for Windows/Mac, so "recommended" seems sufficient.